File: app/src/main-old/java/com/example/santanderapp/santander/detailScreen/interfaceService/StatementsService.java
package com.example.santanderapp.santander.detailScreen.interfaceService;


import com.example.santanderapp.santander.detailScreen.model.ResponseStatement;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;


public interface StatementsService {

    public static final String BASE_URL = "https://bank-app-test.herokuapp.com/api/";

    @GET("statements/{userId}")
    Call<ResponseStatement> listStat(@Path("userId") String userId);

}

File: app/src/main/java/com/joaoneto/testeandroidv2/mainscreen/service/StatementsService.kt
package com.joaoneto.testeandroidv2.mainscreen.service

import com.joaoneto.testeandroidv2.mainscreen.model.StatementsResponseModel
import retrofit2.Call
import retrofit2.http.GET

interface StatementsService {
    @GET("statements/1")
    fun getStatements(): Call<StatementsResponseModel>
}
